src/ManageCourses.ApiClient/ManageCoursesApiClientBase.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace GovUk.Education.ManageCourses.ApiClient
+{
+    public class ManageCoursesApiClientBase
+    {
+        protected readonly IManageCoursesApiClientConfiguration ApiClientConfiguration;
```

Comment:
`ApiClientConfiguration` vs `_apiClientConfiguration`

Reply:
Converted to a protected property (protected fields are frowned upon), which by convention is cased like this